drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -74,6 +74,8 @@ dmnCheatSheet15=ranges
 dmnCheatSheet16=lists
 dmnCheatSheet17=An empty cell is skipped from the evaluation.
 dmnCheatSheet18=In EXPECT section the type could also just a boolean with the result of the assertion and you can use ? identifier to access value to check.
+dmnCheatSheet19=Days and time duration
+dmnCheatSheet20=Years and months duration
```

Comment:
Can you please mention `ISO 8601`? It is the accepted format

Reply:
@danielezonca I mentioned it for `date()` and time()` as well. The result:
![Screenshot from 2020-05-13 11-46-54](https://user-images.githubusercontent.com/16005046/81797844-92503700-950f-11ea-90ef-0e77f1e9cb09.png)